scapy/contrib/automotive/uds_scan.py

```diff
@@ -320,6 +367,156 @@ def _get_table_entry(tup):
                 label)
 
 
+class UDS_RDBISelectiveEnumerator(StagedAutomotiveTestCase):
+    @staticmethod
+    def __connector_random_to_sequential(rdbi_random, _):
```

Comment:
well... that was actually not the idea of the connector. You get the previous enumerator here, and this function should no compute the scan_range for the next Enumerator